wifiphisher/pywifiphisher.py

```diff
@@ -318,14 +327,15 @@ def target_APs():
 
         print ((G + '{0:2}' + W + ' - {1:2}  - ' +
                 T + '{2:{width}} ' + W + ' - ' +
-                B + '{3:17}' + W + ' - {4:12} - ' +
-                R + ' {5:}' + W
+                B + '{3:17}' +  W  + ' - {4:12} - ' +
+                R + ' {5:20}' + B +  ' - {6:3} '
```

Comment:
To get the percentage, you will have to do `2 * (dBm + 100)` where dBm is the RSSI value you retrieved.